totem/circles/mobile_api.py

```diff
@@ -157,14 +160,13 @@ def get_spaces_summary(request: HttpRequest):
             attendees=user,
             circle__published=True,
             cancelled=False,
-            start__gte=timezone.now(),
         )
```

Comment:
Need the start_gte here to no load all events ever made. timezone.now() - datetime.timedelta(minutes=6)